aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1659,7 +1659,7 @@ static at::Tensor _quantized_convolution_onednn(
   ideep::convolution_forward::prepare(
       params, src, packed_weight, expected_bias, dst_dims, dst,
       stride.vec(), dilation.vec(), padding.vec(), padding.vec(), groups,
-      src_scales, weights_scales, ideep::scale_t(1, inv_output_scale),
+      src_scales, weights_scales, ideep::scale_t(1, 1.0f / inv_output_scale),
```

Comment:
Is it a bug fix or what?

Reply:
No, it's changed as expected. Previously, we do the quant decomposition, there is a node as `inv_scale = 1 / scale` from decomposition which will be constant folded. So, we get the `inv_scale` actually.

Reply:
Then the variable name sounds confusing... It is still `inv_output_scale` or `output_scale`?

Reply:
Stack PR: https://github.com/pytorch/pytorch/pull/124246 to change the name to `output_scale`.